recipes/gatk4/meta.yaml

```diff
@@ -12,67 +12,99 @@ source:
 
 build:
   noarch: generic
-  number: 0
+  number: 1
   run_exports:
      - {{ pin_subpackage('gatk4', max_pin="x") }}
 
-requirements:
-  host:
-    - r-base =3.6
-    - r-gplots
-    - r-ggplot2
-    - r-gsalib
-    - r-reshape
-    - gcnvkernel =0.8
-  run:
-    - openjdk >=17,<18
-    - python
-    - r-base =3.6
-    - r-gplots
-    - r-ggplot2
-    - r-gsalib
-    - r-reshape
-    - gcnvkernel =0.8
-
 test:
   commands:
+    - java -version
     - gatk -h
     - gatk --list
     - gatk HaplotypeCaller --help
-    - gatk DetermineGermlineContigPloidy --help
+  imports:
+    - gcnvkernel
 
+#All packages are listed below following GATK's template, except for tensorflow. 
+#The specified version 1.15.0 of tensorflow results in an issue as described in this link: https://github.com/bioconda/bioconda-recipes/issues/45600.outputs:
 outputs:
   - name: gatk4
     script: build_main.sh
     requirements:
+      build:
+        - conda-forge::gcc_impl_linux-64=7.5.0=habd7529_20
+        - conda-forge::gcc_linux-64=7.5.0=h47867f9_33
+        - conda-forge::gfortran_impl_linux-64=7.5.0=h56cb351_20
+        - conda-forge::gfortran_linux-64=7.5.0=h78c8a43_33
+        - conda-forge::gxx_impl_linux-64=7.5.0=hd0bb8aa_20
+        - conda-forge::gxx_linux-64=7.5.0=h555fc39_33
+        - conda-forge::glib=2.66.3=h58526e2_0
+        - conda-forge::python=3.6.10=h8356626_1011_cpython
+        - unzip
+        - wget 
+      host:
+        - absl-py=0.15.0
+        - conda-forge::pip=21.3.1
+        - conda-forge::mkl=2019.5
+        - conda-forge::mkl-service=2.3.0
+        - conda-forge::joblib=1.1.1
+        - conda-forge::numpy=1.17.5
+        - conda-forge::theano=1.0.4=py36h831f99a_1002
+        - defaults::tensorflow=1.15.0=mkl_py36*
+        - conda-forge::scipy=1.0.0
+        - conda-forge::pymc3=3.1
+        - conda-forge::h5py=2.10.0
+        - defaults::_tflow_select=2.3.0=mkl
```

Comment:
please do not use defaults channel ... maybe this is actually the error that you are seeing. I think we do not use the default channel anymore - only conda-forge

Reply:
ah.. thought so. Problem is tensorflow compiled with intel's mkl libraries, which is required by gatk is only available in the defaults channel :(

Reply:
> I think we do not use the default channel anymore - only conda-forge

Is this the case? I thought that the [Bioconda homepage](https://bioconda.github.io/index.html) explicitly says to use `defaults`:

> ## Usage
> First, [install conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
> 
> Then perform a one-time set up of Bioconda with the following commands. This will modify your ~/.condarc file:
> 
> ```bash
> conda config --add channels defaults
> conda config --add channels bioconda
> conda config --add channels conda-forge
> conda config --set channel_priority strict
> ```